test/debezium-avro/mzcompose.yml

```diff
@@ -0,0 +1,134 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+version: '3.7'
+
+mzworkflows:
+  debezium-avro:
```

Comment:
I like the idea of calling this `testdrive` to signify that it's running the testdrive service and it keeps the behavior consistent with `test/testdrive`. What do you think?

Reply:
Judging from ```pipeline.template.yml``` it seems that the convention is that the name of the workflow is identical to the name of the directory where the test is hosted and to the ```id``` field in ```pipeline.template.yml```. So I think ```debezium-avro``` is more appropriate here.